api/management/v1/rds.proto

```diff
@@ -122,35 +121,9 @@ message AddRDSResponse {
   inventory.v1.MySQLService mysql = 3;
   inventory.v1.MySQLdExporter mysqld_exporter = 4;
   inventory.v1.QANMySQLPerfSchemaAgent qan_mysql_perfschema = 5;
+  inventory.v1.PostgreSQLService postgresql = 6;
+  inventory.v1.PostgresExporter postgresql_exporter = 7;
+  inventory.v1.QANPostgreSQLPgStatementsAgent qan_postgresql_pgstatements = 8;
   // Actual table count at the moment of adding.
-  int32 table_count = 6;
-  inventory.v1.PostgreSQLService postgresql = 7;
-  inventory.v1.PostgresExporter postgresql_exporter = 8;
-  inventory.v1.QANPostgreSQLPgStatementsAgent qan_postgresql_pgstatements = 9;
-}
-
-// RDS service provides Node and Service Discovery APIs.
-service RDSService {
-  // DiscoverRDS discovers RDS instances.
-  rpc DiscoverRDS(DiscoverRDSRequest) returns (DiscoverRDSResponse) {
-    option (google.api.http) = {
-      post: "/v1/management/RDS/Discover"
-      body: "*"
-    };
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
-      summary: "Discover RDS"
-      description: "Discovers RDS instances."
-    };
-  }
-  // AddRDS adds RDS instance.
-  rpc AddRDS(AddRDSRequest) returns (AddRDSResponse) {
-    option (google.api.http) = {
-      post: "/v1/management/RDS/Add"
-      body: "*"
-    };
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
-      summary: "Add RDS"
-      description: "Adds RDS instance."
-    };
-  }
+  int32 table_count = 9;
```

Comment:
Maybe it makes sense to have this filed in the begining of the message with num `1` or have a gap and use number `20` for example. Just for case when there will be new RDS services?

Reply:
I remember that we discussed moving table_count to MySQLdExporter

Reply:
Yes, it's in there as well 

```
message AddMySQLResponse {
  inventory.v1.MySQLService service = 1;
  inventory.v1.MySQLdExporter mysqld_exporter = 2;
  inventory.v1.QANMySQLPerfSchemaAgent qan_mysql_perfschema = 3;
  inventory.v1.QANMySQLSlowlogAgent qan_mysql_slowlog = 4;
  // Actual table count at the moment of adding.
  int32 table_count = 5;
}
```

Reply:
not sure about the code you attached, because `AddMySQLResponse != MySQLdExporter`

Reply:
Ah, right - it's not the same. Let me see...

Reply:
I think what you remember from one of the previous rounds of API refactoring has been addressed - https://github.com/percona/pmm/blob/d3ad3cc1bfc5e1833a111b7e07e24d71caf12c8b/api/inventory/v1/agents.proto#L137

This, however, may be also valid, so I suggest we address it in the next round so keep making progress.

Reply:
since we already migrated that into MySQLdExporter, let's drop table_count from other messages